web/script.js

```diff
@@ -0,0 +1,28 @@
+function handleNotifications(title, body, icon, roomid) {
+    if ('Notification' in window) {
+        Notification.requestPermission().then(function(permission) {
+            if (permission === 'granted') {
+                var notification = new Notification(title, {
+                    body: body,
+                    icon: icon,
+                });
+
+                notification.onclick = function() {
+                    console.log('JsFunction::handleNotifications(): On click notification');
+                    var host = window.location.host;
+                    var protocol = window.location.protocol;
+                    var redirectURL = protocol + '//' + host + '/#/rooms/' + roomid;
```

Comment:
do not work on github page and beta twake
https://linagora.github.io/twake-on-matrix/971
better is callback flutter channel

Reply:
In dart code we can't handle it. Only handle it when using Js code